lib/oper1.g

```diff
@@ -601,6 +601,25 @@ BIND_GLOBAL( "INSTALL_METHOD",
       fi;
     fi;
 
+    if IS_FUNCTION(method) and not HAS_NAME_FUNC(method) then
+        funcname := SHALLOW_COPY_OBJ(NAME_FUNC(opr));
+        if info <> false then
+            APPEND_LIST_INTR(funcname, " ");
+            APPEND_LIST_INTR(funcname, info);
```

Comment:
Is there any merit in appending " method" or something similar for the case where info = false? Not sure.